VictoriaAlbert/APIHelper.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  APIHelper.swift
+//  VictoriaAlbert
+//
+//  Created by Marty Avedon on 11/10/16.
+//  Copyright © 2016 Marty Hernandez Avedon. All rights reserved.
+//
+
+import Foundation
+
+//this is modified from my instaCats project: https://github.com/jerjunkel/AC3.2-NSURLSession/blob/master/InstaDogFactory.swift
+
+func APIHelper(endpoint: String, callback: @escaping ([BricABrac]?) -> ()) {
```

Comment:
Let's move over to the class-based APIManager. It works much in the same way as this.